### ISSUE TITLE
Separate PDF's and Webpages

### ISSUE THREAD

**Issue by dpffxhad** [burtonator/polar-bookshelf#533]
NOTE: Polar is supported by community donations.  Bug reports and features are prioritized for people who make a donation: https://opencollective.com/polar-bookshelf

**Is your feature request related to a problem? Please describe.**
It's time consuming to go though all the documents to find an archived webpage.

**Describe the solution you'd like**
An easy way to filter between PDFs and archived web pages, Group PDFs and archived webpages. A menu to select between pdfs and webpages. It'd be nice to have archived web pages separately.

**Describe alternatives you've considered**

**Additional context**

**Comment by burtonator** [comment]
I might add support for 'hidden' tags so you could do something like ```doctype:web```

**Comment by burtonator** [comment]
not in in the short term though.. probably use a tag for now.

**Comment by dpffxhad** [comment]
Can you auto tag "webpage" when we try to capture or use extension?

**Comment by burtonator** [comment]
Yes... but I want to support a type of 'hidden' tag for these otherwise the tags will clutter the UI.  We have support for it , just not implemented yet.

**Comment by dpffxhad** [comment]
Perhaps a **collection** ?  #544
